### PR TITLE
get idaho chip by bbox

### PR DESCRIPTION
**Pull request by naldeborgh7575**
thanks guys!

### CODE REVIEW

**Comment by nricklin** [comment]
please get unit tests working.

File: gbdxtools/idaho.py [hunk 1]
```diff
@@ -221,14 +221,70 @@ def get_idaho_chip(self, bucket_name, idaho_id, center_lat, center_lon,
...
+               str(high_cutoff) + '&format=' + str(format) + '&token=' + access_token)
+
+        if pan_id:
+            url += '&panId='+ pan_id
+
+        r = requests.get(url)
+
+        if r.status_code == 200:
+            # form output path
+            file_path = os.path.join(output_folder, idaho_id+'.tif')
```
Comment (nricklin) [inline gbdxtools/idaho.py]: what happens if I try to grab multiple chips from the same idaho ID?  looks like they'll get overwritten.  Perhaps add an optional input arg for filename & path.

**Comment by nricklin** [comment]
Do you mind merging master into your branch one more time, so I can merge the PR?  Thanks!

**Comment by kstamatiou** [comment]
@naldeborgh7575: Do we still need this functionality? If that is the case, let's fix the conflict so we can merge.

**Comment by youngpm** [comment]
@kostasthebarbarian NateMac is building an API to support the UGHLi tiles that lets you specify bbox, projection, pan or ms, and chip size and get the chip back.  Maybe we can just wrap that functionality once he rolls it out?

**Comment by kstamatiou** [comment]
@youngpm Sounds like a plan. I will close this pull request.

**Pull request closed without merging**